tests/test_resource.py

```diff
@@ -716,6 +716,134 @@ def test_time_index_out_of_bounds():
             assert np.allclose(res["data"][:, 0], time + 10)
 
 
```

Comment:
This is annoying but can you make a new (unrelated) test where we create a 5D dataset in an H5 file and then do an extraction and just make sure all of the slicing is done correctly:
```
with Resource(fp) as res:
    arr = res[dset, ..., gid]
    arr = res[dset, :, :, :, :, gid]  # or this, not sure if ... will work

assert np.allclose(arr, truth[..., gid])

Reply:
No problem!